### PR TITLE
Add support to update mount options for file store instance

### PR DESCRIPTION
**Pull request by harshthakkar01**
### Submission Checklist

Please take the following actions before submitting this pull request.

* Fork your PR branch from the Toolkit "develop" branch (not main)
* Test all changes with pre-commit in a local branch [#](https://goo.gle/hpc-toolkit#development)
* Confirm that "make tests" passes all tests
* Add or modify unit tests to cover code changes
* Ensure that unit test coverage remains above 80%
* Update all applicable documentation
* Follow Cloud HPC Toolkit Contribution guidelines [#](https://goo.gle/hpc-toolkit-contributing)

### CODE REVIEW

File: modules/file-system/filestore/README.md [hunk 1]
```diff
@@ -26,6 +26,21 @@ creation will fail when you run `terraform apply`.
 
 [tiers]: https://cloud.google.com/filestore/docs/service-tiers
 
+### Filestore mount options
+Toolkit uses [default mount options](https://linux.die.net/man/8/mount) for all tiers
+services. Filestore recommends service specific mount options for better performance.
```
Comment (cboneti) [inline modules/file-system/filestore/README.md:31]: s/for all tiers services/for all firestore tiers/

**Review by cboneti — CHANGES_REQUESTED** [review]
Do you think this is enough documentation?

**Review by nick-stroud — APPROVED** [review]

File: modules/file-system/filestore/README.md (hunk 1 above)
Comment (harshthakkar01) [reply modules/file-system/filestore/README.md]: I reworded the documentation. I am not sure if we should put details like what are the recommended options for which tiers. We have a pointer to the recommended options from filestore. Here I only specified how customers can pass this parameter to update the default mount options. If they want to use different file system they can refer the filestore documentation and pass the tier specific recommended option in similar way.

LMKWYT.

Comment (cboneti) [reply modules/file-system/filestore/README.md]: I would still replace "all tier services" for "all filestore tiers"

Comment (harshthakkar01) [reply modules/file-system/filestore/README.md]: Done. Updated "all filestore tiers" to "all tier services"

**Comment by harshthakkar01** [comment]
> Do you think this is enough documentation?

Had an offline discussion with Nick. Updated as per recommendation.

**Review by cboneti — APPROVED** [review]

**Pull request merged**